test/fixtures/parser/tsql/stored_procedure_single_statement.sql

```diff
@@ -35,35 +35,5 @@ BEGIN
        VWSSTAGE.ECDC_CASES
     WHERE DATE_LAST_INSERTED = (SELECT MAX(DATE_LAST_INSERTED) from VWSSTAGE.ECDC_CASES)
 
-    INSERT INTO VWSINTER.ECDC_CASES_OLD
-    (
-        [COUNTRY],
-        [COUNTRY_CODE],
```

Comment:
Doesn't look like this really parsed entirely correctly, but still a shame to lose a test case that presumably was passing. Should this be copied into its own `insert_into_statement.sql` file?